src/Translatable/Translatable.php

```diff
@@ -260,6 +261,40 @@ public function fill(array $attributes)
     }
 
     /**
+     * Get the fillable attributes for the model.
+     *
+     * @return array
+     */
+    public function getFillable()
+    {
+        $fillable = parent::getFillable();
+        $locales = $this->getLocales();
+
+        foreach ($this->translatedAttributes as $attribute) {
+            if ($this->alwaysFillable() || $this->isFillable($attribute)) {
+                foreach ($locales as $locale) {
+                    $fillable[] = $this->getFillableLocalizedAttributeKey($attribute, $locale);
+                }
+                unset($fillable[array_search($attribute, $fillable)]);
```

Comment:
I created this PR in a non-breaking way. Since you're the maintainer, it is completely up to you if this should be breaking (by using `translation.name`) or non-breaking (by still using `name`).

> I don't like the fact we're looping through the locales here.

Well, I'm afraid it's the only way to have Eloquent's `getFillable()` method instantly plugged into `$request->only()`...